digicampipe/tests/test_event_stream.py

```diff
@@ -17,25 +17,3 @@ def test_event_source_new_style():
     for _ in event_stream(example_file_path):
         pass
 
-
```

Comment:
These tests should still be performed on `   'example_100_evts.000.fits.fz' ` or can they ?

Reply:
Tests are done on both.
It is my bad: I copied digicampipe/tests/test_event_stream.py to digicampipe/tests/test_event_stream2.py and modified the original.